packages/nextra/src/server/compile.ts

```diff
@@ -46,6 +46,7 @@ import {
   remarkStructurize
 } from './mdx-plugins/index.js'
 import { rehypeExtractTocContent } from './mdx-plugins/rehype-extract-toc-content.js'
+import { rehypeReactMathjax } from './mdx-plugins/rehype-mathjax-react.js'
```

Comment:
```suggestion
import { rehypeBetterReactMathJax } from './mdx-plugins/rehype-better-react-mathjax.js'
```
Let’s rename to above to be consistent with the used dependency better-react-mathjax